packages/rum/src/domain/record/utils.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { isIE } from '@datadog/browser-core'
+import { getPathToNestedCSSRule } from './utils'
+
+const firstStyleRule = '.selector-1 { color: #aaa }'
+const secondStyleRule = '.selector-2 { color: #bbb }'
+const firstsecondMediaRule = `
+    @media cond-1 {
+        .selector-3-1 { color: #ccc }
+        .selector-3-2 { color: #ddd }
+        .selector-3-3 { color: #eee }
+    }`
+const secondMediaRule = `
+    @media cond-2 {
+        @media cond-2-1 {.selector-2-1-1 { display: none }}
+        @media cond-2-2 {.selector-2-2-1 { display: clock }}
+    }`
+
+describe('getPathToNestedCSSRule', () => {
+  let styleSheet: CSSStyleSheet
+  let styleElement: HTMLStyleElement
+  beforeEach(() => {
+    if (isIE()) {
+      pending('IE not supported')
+    }
+    styleElement = document.createElement('style')
+    document.head.appendChild(styleElement)
+    styleSheet = <CSSStyleSheet>styleElement.sheet
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
    styleSheet = styleElement.sheet!
```